tools/cldr-apps/js/src/esm/cldrGui.mjs

```diff
@@ -1,6 +1,7 @@
 /*
  * cldrGui: encapsulate GUI functions for Survey Tool
  */
+import * as cldrAjax from "./cldrAjax.mjs";
```

Comment:
changes in this file are only for refactoring with makePostData, to encapsulate POST, headers, stringify, and to avoid "await" warning